src/Runtime/Runtime/Core/Rendering/INTERNAL_HtmlDomManager.cs

```diff
@@ -1304,19 +1303,12 @@ static bool IsNumericType(object obj)
 #if OPENSILVER
         internal static void SetVisualBounds(INTERNAL_HtmlDomStyleReference style, Rect visualBounds, bool bSetPositionAbsolute, bool bSetZeroMargin, bool bSetZeroPadding)
         {
-            string position = bSetPositionAbsolute ? "element.style.position=\"absolute\";" : "";
-            string margin = bSetZeroMargin ? "element.style.margin=\"0\";" : "";
-            string padding = bSetZeroPadding ? "element.style.padding=\"0\";" : "";
-            string javaScriptCodeToExecute = $@"
-var element = document.getElementByIdSafe(""{style.Uid}"");
-if (element)
-{{
-element.style.left = ""{visualBounds.Left.ToString(CultureInfo.InvariantCulture)}px"";
-element.style.top = ""{visualBounds.Top.ToString(CultureInfo.InvariantCulture)}px"";
-element.style.width = ""{visualBounds.Width.ToString(CultureInfo.InvariantCulture)}px"";
-element.style.height = ""{visualBounds.Height.ToString(CultureInfo.InvariantCulture)}px"";
-{position}{margin}{padding}
-}}";
+            string left = $"{visualBounds.Left.ToString("0.##", CultureInfo.InvariantCulture)}";
+            string top = $"{visualBounds.Top.ToString("0.##", CultureInfo.InvariantCulture)}";
+            string width = $"{visualBounds.Width.ToString("0.##", CultureInfo.InvariantCulture)}";
+            string height = $"{visualBounds.Height.ToString("0.##", CultureInfo.InvariantCulture)}";

```

Comment:
Add a ToInvariantString(string format) extension method to the OpenSilver.Internal.ObjectExtensions class
```suggestion
            string left = $"{visualBounds.Left.ToString("0.##", CultureInfo.InvariantCulture)}";
            string top = $"{visualBounds.Top.ToString("0.##", CultureInfo.InvariantCulture)}";
            string width = $"{visualBounds.Width.ToString("0.##", CultureInfo.InvariantCulture)}";
            string height = $"{visualBounds.Height.ToString("0.##", CultureInfo.InvariantCulture)}";
```